datadog_checks_dev/datadog_checks/dev/tooling/commands/release/make.py

```diff
@@ -71,6 +71,12 @@ def make(ctx, checks, version, initial_release, skip_sign, sign_only, exclude):
     if initial_release:
         version = '1.0.0'
 
+    repo_choice = ctx.obj['repo_choice']
+
+    # Signing is done by a pipeline in a separate commit
+    if repo_choice != 'core' and not sign_only:
```

Comment:
I'm probably missing some background, but why does it make sense to be able to run a `make --sign-only` on `extras`?

Reply:
`--sign-only` would be used only by the signer bot

Reply:
My assumption here is so that the auto-signing bot can leverage ddev directly, correct @ofek ?
Otherwise we would need a separate logic to create the signing JSON files.